samples/frame_generator.cpp

```diff
@@ -329,14 +334,15 @@ static void setup(Engine* engine, View* view, Scene* scene) {
     if (!g_skyboxOn) {
         auto ibl = FilamentApp::get().getIBL();
         if (ibl) ibl->getSkybox()->setLayerMask(0xff, 0x00);
+    } else {
+        g_skybox = Skybox::Builder().color({
+                ((g_clearColor >> 16) & 0xFF) / 255.0f,
+                ((g_clearColor >>  8) & 0xFF) / 255.0f,
+                ((g_clearColor      ) & 0xFF) / 255.0f,
+                1.0f
+        }).build(*engine);
```

Comment:
Shouldn't iblLuminance be set to 1 in this case?

Reply:
probably? :)